src/Database/Adapter/Mongo.php

```
@@ -0,0 +1,2216 @@
+
+    public function getInternalIndexesKeys(): array
+    {
+        return [];
+    }
+
+    public function getSchemaAttributes(string $collection): array
+    {
+        return [];
+    }
+
+    public function getTenantQuery(string $collection, string $parentAlias = ''): string
```

PHPMD (2.15.0)

Avoid unused parameters such as '$collection'. (Unused Code Rules)

(UnusedFormalParameter)